util/libcrypto.num

```diff
@@ -4371,7 +4371,6 @@ SCRYPT_PARAMS_it                        4314	1_1_1	EXIST:EXPORT_VAR_AS_FUNCTION:
 CRYPTO_secure_clear_free                4315	1_1_0g	EXIST::FUNCTION:
 EVP_PKEY_meth_get0                      4316	1_1_1	EXIST::FUNCTION:
 EVP_PKEY_meth_get_count                 4317	1_1_1	EXIST::FUNCTION:
-RAND_poll_ex                            4318	1_1_1	EXIST::FUNCTION:
```

Comment:
Look like a manual edit.
API deletion will be possible in next major release (1.2.0).

Reply:
But RAND_poll_ex has been added only recently to master by @richsalz and has not entered any stable branch yet, as far as I know.

Reply:
RAND_poll_ex() was introduced in 75e2c877650, so I believe it's no problem to remove it again.

Newbie question: I thought libcrypto.num was meant to be edited manually and libcrypto.def is created from it. If this is not the case, what is the normal procedure then? Are the public headers scanned by perl magic?

Reply:
Yes, RAND_poll_ex can be removed.  The file is canned by perl for missing symbols but not generated.  As for whether or not skipped numbers ("holes") are important or not, I think @levitte can answer that.

Reply:
I can renumber the following symbols to remove the hole introduced by the removal if requested.